includes/wikia/DefaultSettings.php

```diff
@@ -1676,6 +1676,11 @@
  */
 $wgDisableWAMOnHubs = false;
 
+/* @name wgTreatAsCorporate
+ * Treat a wiki as if it was a corporate wiki
+ */
+$wgTreatAsCorporate = false;
```

Comment:
I'm just looking at the line you're quoting (pasted beneath) and I think I must make few things clear.
```
$wgEnableWikiaStatsExt = $wgEnableWikiaStatsExt || $wgEnableWikiaHomePageExt || ($wgEnableWikiaHubsV3Ext && $wgDisableWAMOnHubs);
```
I'd say that the new variable has nothing to do with the old ones. Actually my intent is to make those variables independent. You can set `$wgTreatAsCorporate`, so the page is added to the *Corporate Footer*. You can independently set `$wgDisableWAMOnHubs` on hubs (`$wgEnableWikiaHubsV3Ext = true`), so you can swap WAM module for *Wikia Stats* (`$wgEnableWikiaStatsExt`).

What do you think? @bognix @kvas-damian